src/textual/_compositor.py

```diff
@@ -448,7 +448,7 @@ def _assemble_chops(
         segment_lines: list[list[Segment]] = [
             sum(
                 [line for line in bucket.values() if line is not None],
-                start=[],
+                [],
```

Comment:
the `start` param of the sum function can only be specified with a keyword starting from Python 3.8 - as a result this was causing a crash with Python 3.7